app/screens/home/search/results/file_card.tsx

```diff
@@ -41,11 +42,15 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
             shadowOpacity: 0.8,
             elevation: 1,
         },
-        flexColumn: {flexDirection: 'column'},
         flexRow: {flexDirection: 'row'},
         iconContainer: {
             marginHorizontal: 8,
         },
+        textContainer: {
+            flex: 1,
+            flexDirection: 'column',
+            flexGrow: 1,
```

Comment:
Adding `flexGrow` and `flex` is redundant, since `flex: x` implies `flexGrow: x` (and some other more). Up to you (and the expected functionality) whether makes sense to keep `flex` or only keep `flexGrow`.

Reply:
using only `flex: 1` I get 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/7575921/175113232-5567c8ab-89d6-4ab5-bb31-43135758ca08.png">

using only `flexGrow: 1` I get when I have a super long file name.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/7575921/175113335-000f21f8-f505-44ef-94d1-828b4a828aea.png">

having both produces the following in this case.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/7575921/175113432-5058b5d5-6b2f-491f-92c5-700d09c70233.png">